File: cfgs/java_cfgs_foo/MIDDLE_OF_THREE_USING_MINIMUM_COMPARISONS_1.java
Nodes
1;special;;;
2;conditional;foo_1 > foo_2;14;
3;conditional;foo_1 > foo_2;15;
4;statement;return foo_1;15;
5;exit;;;
6;conditional;foo_1 > foo_2;16;
7;statement;return foo_1;16;
8;statement;return foo_1;17;
9;conditional;foo_1 > foo_2;20;
10;statement;return foo_1;20;
11;conditional;foo_1 > foo_2;21;
12;statement;return foo_1;21;
13;statement;return foo_1;22;
Edges
0;2;
1;3;9;
2;4;6;
3;5;
4;
5;7;8;
6;5;
7;5;
8;10;11;
9;5;
10;12;13;
11;5;
12;5;

File: cfgs/pythoncfgs/MIDDLE_OF_THREE_USING_MINIMUM_COMPARISONS_1.py
Nodes
1;conditional;foo_1 > foo_2;8;
2;conditional;foo_1 > foo_2;16;
3;conditional;foo_1 > foo_2;18;
4;statement;return foo_1;21;
5;statement;return foo_1;19;
6;statement;return foo_1;17;
7;conditional;foo_1 > foo_2;9;
8;conditional;foo_1 > foo_2;11;
9;statement;return foo_1;14;
10;statement;return foo_1;12;
11;statement;return foo_1;10;
12;exit;;
Edges
0;2;7;
1;3;6;
2;4;5;
3;12;
4;12;
5;12;
6;8;11;
7;9;10;
8;12;
9;12;
10;12;
11;
